sdcm/cluster_k8s/__init__.py

```diff
@@ -2593,10 +2593,6 @@ def connection_bundle_file(self) -> Path | None:
     def node_setup(self, node: BaseScyllaPodContainer, verbose: bool = False, timeout: int = 3600):
         if self.test_config.BACKTRACE_DECODING:
             node.install_scylla_debuginfo()
```

Comment:
This file change looks like a `must be a separate PR`.

Reply:
it is...

Reply:
It is still here...

Reply:
true, I read it as `separate commit` - now I see you meant `PR`